src/components/FiatValueDetails/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from 'styled-components'
+import { TYPE } from '../../theme'
+import { CurrencyAmount, JSBI, Percent } from '@swapr/sdk'
+import { warningFiatSeverity } from '../../utils/prices'
+
+interface FiatValueDetailsProps {
+  fiatValue?: CurrencyAmount | null
```

Comment:
Should be required or default to `0` in component properties

Reply:
Should `fiatValue` be computed to a string in the parent component, so we can have `0` as default in component properties? In the other scenario I think having 3 different valid types is a little bit messy, like `CurrencyAmount | string | null`